internal/provider/provider.go

```diff
@@ -360,9 +520,8 @@ func (p *pingfederateProvider) Configure(ctx context.Context, req provider.Confi
 	clientConfig.UserAgentSuffix = pointers.String(userAgentSuffix)
 	resourceConfig.ApiClient = client.NewAPIClient(clientConfig)
 	resp.ResourceData = resourceConfig
-	resp.DataSourceData = resourceConfig
-
 	tflog.Info(ctx, "Configured PingFederate client", map[string]interface{}{"success": true})
+	resp.DataSourceData = resourceConfig
```

Comment:
Any reason this was moved down?

Reply:
zero. I was doing a multi-line edit and this got moved and I didn't realize it was out of place.. will move this back